tpm2d/ml.c

```diff
@@ -124,6 +131,50 @@ ml_get_ima_ml_string_list_new(void)
 	return ima_list;
 }
 
+uint8_t *
+ml_get_measurement_list_binary(size_t *size)
```

Comment:
usually we have the convention to suffix functions which allocates mem for the return value with "_new":
ml_get_measurement_list_binary() -> ml_get_measurement_list_binary_new()

Reply:
Done